samples/matter/lock/README.rst

```diff
@@ -98,13 +99,13 @@ Device Firmware Upgrade support
 .. matter_door_lock_sample_build_with_dfu_start
 
 .. note::
-   Over-the-air Device Firmware Upgrade can be enabled only on hardware platforms containing external flash memory.
+   You can enable over-the-air Device Firmware Upgrade only on hardware platforms that have external flash memory.
 
-You can configure the sample to use the secure bootloader for performing over-the-air Device Firmware Upgrade using Bluetooth® LE, using the ``-DBUILD_WITH_DFU=1`` build flag during the build process.
+To configure the sample to use the secure bootloader for performing over-the-air Device Firmware Upgrade over Bluetooth LE, use the ``-DBUILD_WITH_DFU=1`` build flag during the build process.
```

Comment:
> **over**-the-air Device Firmware Upgrade **over** Bluetooth LE
>
Is repeating the "over" word here correct?

Reply:
Well, it is not totally wrong, at least. In the previous version, it was "using" and then repeated the same word again. It's more a matter of taste.